python/graphstorm/dataloading/sampler.py

```diff
@@ -210,3 +210,47 @@ def _gen_neg_pair(pos_pair, utype, vtype):
             pos_neg_tuple = _gen_neg_pair(pos_pairs,
                 g.canonical_etypes[0][0], g.canonical_etypes[0][2])
         return pos_neg_tuple
+
+class JointLocalUniform(JointUniform):
+    '''Jointly corrupt a group of edges.
+
+    The main idea is to sample a set of nodes and use them to corrupt all edges in a mini-batch.
+    This algorithm won't change the sampling probability for each individual edge, but can
```

Comment:
i don't understand this statement. it changes the sampling probability for the negative nodes. why does it not change the sampling probability for each edge?

Reply:
It does not change the probability of positive edges. Let me update the docstr.